connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2726,6 +2750,7 @@ class SparkConnectPlanner(val sessionHolder: SessionHolder) extends Logging {
     SparkConnectService.streamingSessionManager.registerNewStreamingQuery(
       sessionHolder = SessionHolder(userId = userId, sessionId = sessionId, session),
       query = query)
+    executeHolder.events.postFinished()
```

Comment:
@rangadi other streaming commands will just be posting a "started" and "finished" event, with no insight inside them

Reply:
What is this? If it is just post of RPC, can we do this in higher in the call stack?
Also, it won't be posted if there is an error above this (e.g. query fails to start).

Reply:
finished, canceled & failed are all terminal states
(i.e. there is no finished after failed)

Reply:
Where is failed sent? Better to move finished also there?

Reply:
@rangadi any exception thrown will go to ErrorUtils.handleError; that's where failed is sent.

Reply:
@rangadi can there be an error that will not throw and exception?

Reply:
I don't think so. There should be an exception otherwise, a query id is returned. 

> any exception thrown will go to ErrorUtils.handleError; that's where failed is sent.

Hope there is only one place for success event as well like this. Better not to repeat this code in many places like this.